DOCS/man/input.rst

```diff
@@ -1674,9 +1674,9 @@ Property list
     card is performed. Can also be used for channel switching.
 
 ``sub-text``
-    Return the current subtitle text. Formatting is stripped. If a subtitle
-    is selected, but no text is currently visible, or the subtitle is not
-    text-based (i.e. DVD/BD subtitles), an empty string is returned.
+    Return the current subtitle text. Formatting is stripped. If the 
```

Comment:
the fact that "invisible" subtitles might get returned deserves mentioning

Reply:
Could you clarify what you mean by that?

Reply:
I would consider it surprising that `sub-text` returns subtitles that aren't actually visible (`sub-visibility=no`), the docs should mention that.

Reply:
Ah, that makes sense.